sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOptions.cs

```diff
@@ -16,5 +16,17 @@ public class AnalyzeHealthcareEntitiesOptions : TextAnalyticsRequestOptions
         public AnalyzeHealthcareEntitiesOptions()
         {
         }
+
+        /// <summary>
+        /// Optional display name for the operation.
+        /// </summary>
+        public string DisplayName { get; set; }
+
+        internal new AnalyzeHealthcareEntitiesOptions CheckSupported(TextAnalyticsClientOptions.ServiceVersion version)
+        {
+            base.CheckSupported(version);
+            Validation.SupportsProperty(DisplayName, nameof(DisplayName), TextAnalyticsClientOptions.ServiceVersion.V2022_05_01, version);
```

Comment:
Should we use an `Optional<string>` backing field to tell if the property was ever set instead of just a `null` check?

Reply:
Good idea. I'll look into that. Seems plausible.

Reply:
Per offline conversation, I'm going to punt some of this to a lower-pri issue and will try to get to it before GA, but I will make a few of these changes - especially around messaging - as part of this PR. Implementation details beyond messaging can wait, since there's little time before GA.

Reply:
Actually, checking `Nullable<bool>.HasValue` (or just a null check), which `SupportsProperty` does, is better because that's what the generated `Create{Whatever}Request` does:
```C#
            if (loggingOptOut != null)
            {
                uri.AppendQuery("loggingOptOut", loggingOptOut.Value, true);
            }
```
I feel it's better to be consistent.